src/app/containers/ChartbeatAnalytics/amp/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { string, number } from 'prop-types';
+
+const chartbeatAmpConfigOptions = options => ({
+  vars: {
+    uid: options.chartbeatUID,
+    sections: options.sections,
+    domain: options.domain,
+    contentType: options.type,
+    idSync: {
+      bbc_hid: options.cookie,
+    },
+  },
+});
+
+const JsonInlinedScript = data => (
```

Comment:
We're trying to move away from having to use `dangerouslySetInnerHTML`, can we look into taking the approach used in the original AV implementation? 

https://github.com/bbc/simorgh/blob/0df6465e69241433dabac1a0be71a757380048e1/src/app/components/AudioVideoHead/index.jsx#L23-L47

Reply:
Thanks @rhell, I tried that earlier but I don't think that will work with amp, the only acceptable types for the `script` tag for amp would be `"application/json"` and `"text/plain"` [amp docs](https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#html-tags)